### DISCUSSION TITLE
ReadableSession test utils

### DISCUSSION THREAD

**Discussion by lcmgh** [maxcountryman/axum-sessions#41] [Q&A]
Hi!

Are there any utilities to easily create a `ReadableSession` object for testing purposes?

Thanks

**Comment by maxcountryman** [comment]
Currently that I'm aware of, no.

However I believe async-session is going to be updated soon and when it is will include a change that makes sessions directly clonable.

When that happens we'll move back to using the session without the need for separate extractors.
